languages/nl.php

```diff
@@ -341,8 +372,8 @@
 	'river:none' => 'Geen activiteit',
 	'river:update' => 'Update van %s',
 	'river:delete' => 'Verwijder deze activiteit',
-	'river:delete:success' => 'Het river-item is verwijderd',
-	'river:delete:fail' => 'Het river-item kon niet worden verwijderd',
+	'river:delete:success' => 'Activity item has been deleted',
+	'river:delete:fail' => 'Activity item could not be deleted',
```

Comment:
any idea why this is reverted to english? Dutch translation was present... maybe removed in transifex?

Reply:
Yes, seems to be missing from Transifex for some reason.